### PR TITLE
Add Interval to moment-format

### PR DESCRIPTION
**Pull request by cdnrg**
Added interval parameter to moment-format

### CODE REVIEW

File: addon/helpers/moment-format.js [hunk 1]
```diff
@@ -13,15 +14,25 @@ export default BaseHelper.extend({
...
+    }
+
+    if (interval) {
+      this.timer = setTimeout(runBind(this, this.recompute), parseInt(interval, 10));
     }
 
     let format;
     const args = [];
+    
+    if(params[0] === 'now') {
```
Comment (jasonmit) [inline addon/helpers/moment-format.js]: This change is out of scope but I'd be happy to consider it in the future just not in this changeset

**Comment by jasonmit** [comment]
Just needs to be covered with some tests and the removal of `now`.  Thanks :+1:

**Comment by jasonmit** [comment]
https://github.com/stefanpenner/ember-moment/pull/151 fixes this by adding interval to all helpers

**Comment by cdnrg** [comment]
Thanks!

**Pull request closed without merging**